packages/core/admin/admin/src/components/UnstableGuidedTour/Tours.tsx

```diff
@@ -203,7 +201,7 @@ const tours = {
     {
       name: 'Fields',
       content: (Step) => (
-        <Step.Root side={'top'} align="start" sideOffset={-36}>
+        <Step.Root side={'top'} sideOffset={-36}>
```

Comment:
I think it should just be the default side top center and I have a hunch the sideOffset is not necessary anymore but haven't tested yet. 

```suggestion
        <Step.Root>
```